docker-compose.yml

```diff
@@ -119,8 +119,8 @@ services:
       - ./postgres_cfg/db.sql:/docker-entrypoint-initdb.d/db.sql:Z
       - ${PG_DATA_DIR:-./postgresql}:/var/lib/postgresql/data:Z
     command: postgres -c config_file=/etc/postgresql/postgresql.conf
-    ports:
-      - 127.0.0.1:5432:5432
+    ports: []
+    #  - 127.0.0.1:5432:5432
```

Comment:
Is this change done to not expose postgres?

Reply:
Yup, also since many people run into issue because they run postgres locally as well which causes port conflict.